pkg/providers/instancetype/suite_test.go

```diff
@@ -276,7 +280,7 @@ var _ = Describe("Instance Types", func() {
 		}
 
 		// Ensure that we're exercising all well known labels except for gpu labels and nvme
-		expectedLabels := append(v1alpha5.WellKnownLabels.Difference(sets.NewString(
+		expectedLabels := append(v1alpha5.WellKnownLabels.Clone().Delete(v1.LabelWindowsBuild).Difference(sets.NewString(
```

Comment:
Same comment here about putting it in the same bucket as the other labels that we are excluding here